go.mod

```diff
@@ -45,7 +45,7 @@ require (
 	github.com/hetznercloud/hcloud-go v1.57.0
 	github.com/jackpal/gateway v1.0.14
 	github.com/kubermatic/grafanasdk v0.9.13
-	github.com/kubermatic/machine-controller v1.59.2
+	github.com/kubermatic/machine-controller v1.59.3
```

Comment:
I cannot see where this version has anything with Tinkerbell in it. Isn't the whole Tinkerbell stuff in the main branch?